addons/account/models/account_journal_dashboard.py

```diff
@@ -61,13 +61,16 @@ def _get_json_activity_data(self):
         today = fields.Date.context_today(self)
         activities = defaultdict(list)
         # search activity on move on the journal
-        sql_query = """
+        lang = self.env.user.lang or get_lang(self.env).code
+        act_name = f"COALESCE(act_type.name->>'{lang}', act_type.name->>'en_US')" \
+            if self.pool['mail.activity.type'].name.translate else 'act_type.name'
```

Comment:
When is it not translatable?

Reply:
In theory never, it's more defensive, in case people do weird things with studio or inherit the field in an incomplete way

Reply:
I'd rather not try to cover all possible unlikely cases and keep the code simple :)

Reply:
Aight, fine by me, changing that